packages/browser/src/plugins/remote-loader/index.ts

```diff
@@ -79,6 +85,7 @@ export class ActionDestination implements DestinationPlugin {
         transformedContext = await this.transform(ctx)
       }
 
+      await this.ready()
```

Comment:
Doesn't this need to happen before the `transform` call above?

Reply:
Not necessary, `transform` is internal function, not a proxy to any of the plugin's function